aiida/engine/processes/builder.py

```diff
@@ -242,3 +259,10 @@ def __init__(self, process_class: Type['Process']):
     def process_class(self) -> Type['Process']:
         """Return the process class for which this builder is constructed."""
         return self._process_class
+
+    def _repr_pretty_(self, p, _) -> str:  # pylint: disable=invalid-name
+        """Pretty representation for in the IPython console and notebooks."""
+
+        return p.text(
+            f'Process class: {self._process_class.get_name()}\nInputs:\n{json.dumps(self, cls=PrettyEncoder, indent=4)}'
```

Comment:
`get_name()` wait what now? What is this method?

Reply:
Umm, I found it when I was trying to get a short version of the process class:

```python
In [1]: from aiida.calculations.arithmetic.add import ArithmeticAddCalculation

In [2]: builder = ArithmeticAddCalculation.get_builder()

In [3]: builder._process_class
Out[3]: aiida.calculations.arithmetic.add.ArithmeticAddCalculation

In [4]: builder._process_class.get_name()
Out[4]: 'ArithmeticAddCalculation'
```

Reply:
Ah, it comes from `plumpy.Process`:
```python
Signature: builder._process_class.get_name() -> str
Source:   
    @classmethod
    def get_name(cls) -> str:
        """Return the process class name."""
        return cls.__name__
File:      ~/code/aiida/env/dev/plumpy/plumpy/processes.py
Type:      method
```
To be honest, I could simply use the builtin `__name__`. See no reason not to